app/ui/browser-modern/actions/page-actions.js

```diff
@@ -110,6 +110,14 @@ export function setPageUIState(pageId, pageUIState) {
   };
 }
 
+export function preventInteractionForPage(pageId) {
```

Comment:
I think enable/disable is more commonly used in the codebase than prevent/allow FWIW